documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -155,7 +157,7 @@ Instead, we need to make the value [_reactive_](https://learn.svelte.dev/tutoria
 </script>
 ```
 
-Reusing components like this means that things like sidebar scroll state are preserved, and you can easily animate between changing values. However, if you do need to completely destroy and remount a component on navigation, you can use this pattern:
+Reusing components like this means that things like sidebar scroll state are preserved, and you can easily animate between changing values. However, because only the state changes all other components on the page will also behave the same and will not remount if they are already visible on the page, only their props will change. In the case that you do need to completely destroy and remount a component on navigation, you can use this pattern:
```

Comment:
This seems a bit verbose.

```suggestion
Reusing components like this means that things like sidebar scroll state are preserved, and you can easily animate between changing values. In the case that you do need to completely destroy and remount a component on navigation, you can use this pattern:
```

How about keeping it as is and addressing child components in the paragraph above?